domains/Thebidouilleur.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "QJoly",
+        "email": "quentinjoly@thoughtless.eu"
+      },
+
+      "record": {
+        "A": ["5.39.75.213"]
```

Comment:
404?

Reply:
I'm using a reverse proxy, by default : it's just a 404. 
If you curl this IP with a custom header, you will have a result :) 

```bashrc
 curl https://5.39.75.213/ -H "Host:git.thoughtless.eu" -k
 ```